function testcase(){try{return eval("_11_13_2_2 /= 1;"),!1}catch(a){return a instanceof ReferenceError}}runTestCase(testcase);
